superset/views/base_api.py

```diff
@@ -217,33 +213,19 @@ class BaseSupersetModelRestApi(ModelRestApi):
     }
 
     def __init__(self) -> None:
+        super().__init__()
         # Setup statsd
         self.stats_logger = BaseStatsLogger()
         # Add base API spec base query parameter schemas
         if self.apispec_parameter_schemas is None:  # type: ignore
             self.apispec_parameter_schemas = {}
         self.apispec_parameter_schemas["get_related_schema"] = get_related_schema
-        if self.openapi_spec_component_schemas is None:
-            self.openapi_spec_component_schemas = ()
-        self.openapi_spec_component_schemas = self.openapi_spec_component_schemas + (
+        self.openapi_spec_component_schemas: Tuple[
+            Type[Schema], ...
+        ] = self.openapi_spec_component_schemas + (
```

Comment:
Do we really need this type declaration? Seems like this should be enough:
```suggestion
        self.openapi_spec_component_schemas = self.openapi_spec_component_schemas + (
```

Reply:
it does, and it surprised me also, this is declared on FAB here: https://github.com/dpgaspar/Flask-AppBuilder/blob/master/flask_appbuilder/api/__init__.py#L297

Reply:
I'm trying to merge master and getting an error on this line.

AttributeError: 'AnnotationRestApi' object has no attribute 'openapi_spec_component_schemas'

Why were lines 226 and 227 removed?
```
        if self.openapi_spec_component_schemas is None:
            self.openapi_spec_component_schemas = ()
```